modules/apps/organizations/organizations-test/src/testIntegration/java/com/liferay/organizations/service/test/OrganizationLocalServiceTest.java

```diff
@@ -547,6 +548,50 @@ public void testHasUserOrganization2() throws Exception {
 			organizationAA.getOrganizationId(), TestPropsValues.getUser());
 	}
 
+	@Test
+	public void testIncompleteOrganization() throws Exception {
+		User user = TestPropsValues.getUser();
+
+		Organization organization = _organizationLocalService.addOrganization(
+			RandomTestUtil.randomString(), user.getCompanyId(),
+			user.getUserId());
+
+		Assert.assertTrue(organization.getBatchImportStatus() == 1);
+
+		List<Organization> organizations =
```

Comment:
is there a getOrganizations where we have only the company? Because in this case we are not sure if it doesn't return because it's incomplete or because doesn't have the parentOrganizationId set